views/js/qtiCommonRenderer/tpl/container.tpl

```diff
@@ -1,5 +1,5 @@
 {{~#equal contentModel "blockStatic"~}}
-<div class="qti-block">{{{body}}}</div>
+<div class="qti-block"><span>{{{body}}}</span></div>
```

Comment:
could we not simply rely on the <div class="qti-block"> to enable the styling ? 
This div has indeed been added to do such a thing.